packages/vx-shape/test/Pie.test.tsx

```diff
@@ -50,11 +50,6 @@ const PieChildren = ({ children, ...restProps }: Partial<PieProps<Datum>>) =>
   );
 
 describe('<Pie />', () => {
-  beforeEach(() => {
-    // eslint-disable-next-line @typescript-eslint/unbound-method
-    global.console.error = jest.fn();
```

Comment:
is this because `nimbus` handles this for us?

Reply:
This `console.error` is one test I am not sure what is going on. It still throw but `console.error` was not called. I also question the need to check if `console.error` is called.